client/src/components/Lists/PitchItems/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, ReactElement } from 'react';
+import { IPitch } from 'ssw-common';
+
+import { FieldTag } from '../..';
+import { useInterests, useTeams } from '../../../contexts';
+
+interface PitchItemsProps {
+  pitch: Partial<IPitch>;
+  type: 'topics' | 'userTeams';
+}
+
+const PitchItems: FC<PitchItemsProps> = ({ pitch, type }): ReactElement => {
```

Comment:
Andrew is writing a really similar component. Might want to cross check both and see which is better. The solution of just pulling the first assignment contributor's teams doesn't actually work. You could be on two teams (one of which is not needed on the pitch).

Reply:
I should probably make this component for only user teams because I use this when assignmentContributors is filtered to include only the user of the profile page, so it wouldn't work for other cases when assignmentContributors isn't filtered. In the profile page contributions table, for the column `teams you're on` we need to get the teams that the user had claimed for the pitch. I thought a way of doing that was from assignment contributors because it stores the teams for each user who is contributing. Wouldn't it have the teams that the user had picked for the pitch?

Reply:
Never mind I'm going to this another way by filtering the user teams later on